### PR TITLE
Update flatten-el8 with creating individual repos per module

### PR DESCRIPTION
**Pull request by pcreech**
This also simplifies deployment

### CODE REVIEW

File: koji/flatten-el8/rhel8-split.sh [hunk 1]
```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 HOMEDIR=/root/koji-sync/podman/
-BINDIR=/usr/local/bin
+BINDIR=/root/koji-sync/podman/
```
Comment (ehelms) [inline koji/flatten-el8/rhel8-split.sh:3]: The BINDIR appears to be the same as the HOMEDIR now

File: koji/flatten-el8/rhel8-split.sh [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+
+dnf install -y createrepo_c libmodulemd librepo python36 python3-hawkey python3-librepo python3-gobject-base "dnf-command(reposync)"
+
+
+pushd /root/koji-sync/podman/repos
```
Comment (ehelms) [inline koji/flatten-el8/rhel8-split.sh:8]:
```suggestion
pushd $HOMEDIR/repos
```

File: koji/flatten-el8/rhel8-split.sh (hunk 1 above)
Comment (pcreech) [reply koji/flatten-el8/rhel8-split.sh:3]: Yes, this is intentional, as the executable is referenced $BINDIR/splitter.py.  (_could_ probably just $HOMEDIR it to minimize redundancy)

File: koji/flatten-el8/test.sh [hunk 3]
```diff
@@ -1,4 +1,3 @@
-podman run --privileged -v /home/koji-sync/podman:/root/koji-sync/podman centos:8 /bin/sh /root/koji-sync/podman/script.sh
-
-
-rsync -r --delete -e ssh /home/koji-sync/podman/koji/staged/ root@koji.katello.org:/mnt/koji/releases/split/yum/koji-modules/koji/staged/ -v
+podman run --privileged -v ./:/root/koji-sync/podman centos:8 /bin/sh /root/koji-sync/podman/rhel8-split.sh
```
Comment (ehelms) [inline koji/flatten-el8/test.sh:1]: Should this file have a shebang as the first line?

Comment (pcreech) [reply koji/flatten-el8/test.sh:1]: Technically, probably.

File: koji/flatten-el8/README.md [hunk 4]
```diff
@@ -1 +1,8 @@
 This is a quick hack I put together from what was done by the epel folks in fedora infrastructure to build packages against modules
+
+Usage:  
+
+	`./test.sh `
```
Comment (ehelms) [inline koji/flatten-el8/README.md:5]: Based on the name of this script, I thought if I ran it things could just execute locally and not make changes over on Koji. Looking at that script it looks like the thing to run to generate updated repositories -- is that accurate? If so, could we rename to better capture that so I don't go and do dumb things :)

Comment (pcreech) [reply koji/flatten-el8/README.md:5]: Yeah, naming could use an update.  The script is intended to be an all in one command.

Comment (ekohl) [reply koji/flatten-el8/README.md:5]: You're mixing 2 styles to make it code in markdown leading to weird rendering:
```suggestion
	./test.sh
```

File: koji/flatten-el8/rhel8-split.sh (hunk 1 above)
Comment (pcreech) [reply koji/flatten-el8/rhel8-split.sh]: this should be fixed now.

File: koji/flatten-el8/rhel8-split.sh (hunk 2 above)
Comment (pcreech) [reply koji/flatten-el8/rhel8-split.sh]: Done

File: koji/flatten-el8/test.sh (hunk 3 above)
Comment (pcreech) [reply koji/flatten-el8/test.sh]: Shebang added.

File: koji/flatten-el8/README.md (hunk 4 above)
Comment (pcreech) [reply koji/flatten-el8/README.md]: Updated to a different name, and removed the back ticks.

**Review by ehelms — APPROVED** [review]

**Pull request merged**